spec/queries/project/subscriptions_pledged_value_query_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Project::SubscriptionsPledgedValueQuery do
+  describe ".call" do
+    let(:subscriptions_pledged_value) { Project::SubscriptionsPledgedValueQuery.new(project) }
+    let(:project) { create(:project) }
+
+    before { allow_any_instance_of(Project).to receive(:recurring?).and_return(true) }
```

Comment:
I have needed to do this because there is a validation when you associate plans with projects that uses this method. IMO, is better simply stub this method than create a recurring channel and associate it to the project.